<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>중고폰나라 자동 포스팅기</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>18</y>
          <width>151</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>작업시작</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>541</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>324</width>
          <height>132</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_8">
           <property name="text">
            <string>게시글 시간설정</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>초</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>작업대기시간:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_9">
           <property name="text">
            <string>반복작업</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>전체대기시간:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>초</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>632</x>
          <y>371</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>작업종료</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>215</width>
          <height>97</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_4">
           <property name="text">
            <string>우클릭허용</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>댓글허용</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>전체알람</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>검색됨</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_5">
           <property name="text">
            <string>연락처 노출 동의</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_6">
           <property name="text">
            <string>일회용 번호 사용</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_7">
           <property name="text">
            <string>네이버 페이</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>470</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>332</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>..</string>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>12</y>
            <width>741</width>
            <height>291</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>739</width>
             <height>289</height>
            </rect>
           </property>
           <widget class="QTextEdit" name="textEdit">
            <property name="geometry">
             <rect>
              <x>53</x>
              <y>71</y>
              <width>671</width>
              <height>192</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>71</y>
              <width>26</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string>내용</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>40</y>
              <width>26</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string>제목</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>57</x>
              <y>40</y>
              <width>661</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>12</y>
              <width>65</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string>게시글작성</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>330</y>
            <width>211</width>
            <height>91</height>
           </rect>
          </property>
          <property name="text">
           <string>글등록</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>330</y>
            <width>221</width>
            <height>88</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>태그</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>판매가</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>329</y>
            <width>221</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>대표이미지</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>카테고리</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>카페</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>370</y>
            <width>381</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>저장</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>삭제</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>전체삭제</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>771</width>
            <height>351</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>시간</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>새 행</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>제목</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>가격</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>대표이미지</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>카테고리</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>태그</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>카페</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>category_id</string>
           </property>
          </column>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
